packages/utilities/psammead-styles/src/fonts.js

```diff
@@ -356,6 +359,38 @@ export const F_LATHA_BOLD = baseUrlOverride => `
   }
 `;
 
+// Tamil
+export const F_TAMIL_REGULAR = baseUrlOverride => `
+  @font-face {
+    font-family: "Tamil";
+    font-weight: 400;
```

Comment:
how was this value arrived at?

Reply:
I think they are supplied by ux as the common weights

Value | Common weight name
-- | --
400 | Normal (Regular)
500 | Medium
600 | Semi Bold (Demi Bold)
700 | Bold

Reply:
400 is the standard weight for regular and 700 is the standard weight for bold :)